README.md

```diff
@@ -1 +1,73 @@
-# hello-microverse
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
+
+> Description the project.
```

Comment:
Also, remember to update the description of the project here.
You could replace `Description the project.` with `This project is about setting up all required tools and standards (e.g. Linters) in a simple "Hello Microverse" project.`